nixos/modules/services/desktops/playerctld.nix

```diff
@@ -0,0 +1,31 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.playerctld;
+in
+{
+  options.services.playerctld = {
+    enable = lib.mkEnableOption ("the playerctld daemon");
+
+    package = lib.mkPackageOption pkgs "playerctl" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    systemd.user.services.playerctld = {
+      description = "Playerctld daemon to track media player activity";
+      wantedBy = [ "multi-user.target" ];
```

Comment:
If you want to install this service as a user service, are you sure you want to use the `multi-user.target`? I only know this target from the system level instance. Does it transparently find, that this is what you want to depend upon? 

When I list the user targets, I see only these:

```
$ systemctl --user list-units --type=target
  UNIT           LOAD   ACTIVE SUB    DESCRIPTION
  basic.target   loaded active active Basic System
  default.target loaded active active Main User Target
  paths.target   loaded active active Paths
  sockets.target loaded active active Sockets
  timers.target  loaded active active Timers
```

Reply:
I think it is better to use default as you mentionned